### DISCUSSION TITLE
Confluent Kafka setup with mTLS (mutual TLS)

### DISCUSSION THREAD

**Discussion by nick-szabadkai** [provectus/kafka-ui#1638] [Q&A] [answered]
Hallo,

i am a new fan of kafkaui. Thanks for let us use it.

I setup Confluent Community Edition with Ansible and mTLS on our VMs. All seperate VMs.

Whe i use mTLS everything is working internally great but i can noch reach out for the zookeeper because it says:
[2022-02-18 15:24:20,448] ERROR Unsuccessful handshake with session 0x0 (org.apache.zookeeper.server.NettyServerCnxnFactory)
[2022-02-18 15:24:20,448] WARN Exception caught (org.apache.zookeeper.server.NettyServerCnxnFactory)
io.netty.handler.codec.DecoderException: io.netty.handler.ssl.NotSslRecordException: not an SSL/TLS record: 0000002d0000000000000000000000000000ea600000000000000000000000100000000000000000000000000000000000

But without SSL it is working.

Do you have an ideas where to search for?
Is it possible for kafkaui to connect a zookeeper with mTLS or not yet?

Thanks
Nick

**Comment by Haarolean** [comment]
Hi, glad you like it.

Please share your kafka-ui config/docker-compose.

**Comment by Haarolean** [comment, marked as answer]
We've dropped support for zookeeper. It's not required for any of the functionality.
